lib/adjustable_income_calculator.rb

```diff
@@ -0,0 +1,99 @@
+class AdjustableIncomeCalculator
+  # https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/lifeexpectancies/bulletins/lifeexpectancyatbirthandatage65bylocalareasinenglandandwales/2015-11-04#national-life-expectancy-at-age-65
```

Comment:
A commit message on the commit that introduced this code would maybe be more suitable for this particular reference?